vuu-ui/packages/vuu-layout/src/layout-persistence/LocalLayoutPersistenceManager.ts

```diff
@@ -171,13 +180,16 @@ export class LocalLayoutPersistenceManager implements LayoutPersistenceManager {
 
   // Ensures that there is exactly one element (Layout or Metadata) in local
   // storage corresponding to the provided ID.
-  private validateId(
+  validateId: (
```

Comment:
this can be rewritten like this: 
```
validateId = (id: string, dataType: "metadata" | "layout"): Promise<void> => {
...body of the function
}
```